.github/workflows/deploy_whole_doc.yml

```diff
@@ -34,13 +36,15 @@ jobs:
 
   deploy_framework:
     name: Deploy framework
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Same here, why not 22

Reply:
latest is not really a proper choice because we never now if it will work when the latest ubuntu change on GitHub, it is a better thing to fix the ubuntu version